requests/test_benchmark_business.json

```diff
@@ -134,5 +134,6 @@
             "url": "/submitted/thank-you/"
         }
     ],
-    "schema_name": "test_benchmark_business"
+    "schema_name": "test_benchmark_business",
+    "schema_url": "https://storage.googleapis.com/eq-questionnaire-schemas/test_benchmark_business.json"
```

Comment:
I think it is worth adding this key value into all our existing schemas in `requests/` folder.

Reply:
Only benchmark business exists in that eq-questionnaire-schemas bucket though?

Reply:
That's true, but our current scripts would be adding this key value in regardless of whether it exists in the bucket.